go/integTest/shared_commands_test.go

```diff
@@ -4242,3 +4242,30 @@ func (suite *GlideTestSuite) TestZRem() {
 		assert.IsType(suite.T(), &api.RequestError{}, err)
 	})
 }
+
+func (suite *GlideTestSuite) TestPersist() {
+	suite.runWithDefaultClients(func(client api.BaseClient) {
+		//Test 1: Check if persist command removes the expiration time of a key.
+		keyName := "{keyName}" + uuid.NewString()
+		suite.verifyOK(client.Set(keyName, initialValue))
+		resultExpire, err := client.Expire(keyName, 300)
+		assert.Nil(suite.T(), err)
```

Comment:
```suggestion
		assert.Nil(t, err)
```
Same for all occurrences of suite.T() in the test.

Reply:
fix in commit f23f301d0b19e2a3b9edbaa619ba73a29cde2a87